lectures/bayes_nonconj.md

```diff
@@ -1,12 +1,10 @@
 ---
 jupytext:
   text_representation:
-    extension: .myst
+    extension: .md
```

Comment:
@kp992 the extension should remain `md` usually. Has this changed in `jupytext`?

Reply:
ah sorry diff was the wrong way around in my head 👍